internal/tools/aws/session_test.go

```diff
@@ -0,0 +1,13 @@
+package aws
+
+// func (a *AWSTestSuite) TestSingletonSession() {
```

Comment:
Please don't check in commented-out code unless there's a very good reason. There's always version control history to fetch old code; leave a comment here if needed.

Reply:
That was an accident haha